public/locales/en/translation.json

```diff
@@ -0,0 +1,8 @@
+{
+    "dashboard": "Dashboard",
+    "order": "Order",
+    "table": "Table",
+    "menu": "Menu",
+    "invoice": "Invoice",
+    "account": "Account"
+}
```

Comment:
Translation adds up pretty quickly. It is more robust to group up translations.

Please consider a following format:
```json
{
  "common": {
    "my_account": "My Account",
    "profile": "Profile",
    "billing": "Billing",
    "team": "Team",
    "logout": "Logout"
  }
}
```
```ts
Example Usage: t("common.my_account")
```
A suggestion from the top of my head would be
- **common** : for translations that appears commonly in every components of the projects.
- **component_name**: for translations that are specific to a components.
- **page_name**: for translations that are specific to a page..

The list would go on, but as of now this could suffice.

Reply:
How about duplicate world on different components or screens  in case the dashboard page has `dashboard ` and on nav menu has `dashboard ` so we need to use:
1. `t("common.dashboard")` in the side dashboard screen 
2. `t("dashboard.dashboard")` because is base on components and screen

In Case 1 it breaks the rule because it uses common in dashboard page
but in case 2 it creates a problem of having duplicate JSON values with a different key


I'm happy to solve it please discuss more that , because I have that problem on my last project

Reply:
My apology for the belated reply.
In case the word dashboard presents many time on different components or views.
`dashboard` in our case can be considered a common translation string.
All the same page should access the translation from `t("common.dashboard")`

Group with {component_name} or {page_name} as based, should contain a translation that only, uniquely use by a single component or page
- {component_name}: for translations that are specific to a components.
- {page_name}: for translations that are specific to a page..

#

Another example is most of the input title, including but not limited to
- Edit
- Delete
- Sign In
- Logout

should remain inside common

whereas, `My Dashboard` title inside a /dashboard page for example, should contain under `dashboard` group.
`Usage: t('dashboard.my_dashboard')`

Reply:
thanks for your detailed explanation i think this comment can be a rule for all contributors. we can put it in contribute.md  (@panhabot)  and follow it could be don't make confused for other contributor.

Reply:
Pleasure!
@vicheanath Please adjust the translation file, so that we can do a final review.
Though everything else seem fine to me. After the changes, the PR should be approved.